### PR TITLE
(DOCSP-10382) PATH note

### PR DESCRIPTION
**Pull request by zach-carr**
[JIRA](https://jira.mongodb.org/browse/DOCSP-10382)
[Staged (commands)](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker/DOCSP-10382/commands#shell-commands)
[Staged (settings)](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker/DOCSP-10382/settings#settings)

Ticket was discussed over Zoom, we decided to include a note about adding the shell to your PATH but not a full set of instructions.

### CODE REVIEW

File: source/includes/admonitions/warn-configure-path.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. important::
+
+   To launch your selected MongoDB shell, you must:
+
+   - Install the shell you want to use, and
+
+   - Configure your machine's ``PATH`` environment variable to include 
+     the filepath to your shell binary.
+
+   If you install a new version of your MongoDB shell, update your
```
Comment (MongoCaleb) [inline source/includes/admonitions/warn-configure-path.rst:10]: > If you install a new version of your MongoDB shell, update your ...
=>
If you install a new version of your MongoDB shell, **you will need to** update your ...

**Comment by zach-carr** [comment]
@mmarcon, this note about installing and adding the shell you want to use to your PATH is ready for you to review whenever you have some time. Thank you!

**Comment by mmarcon** [comment]
I think this looks good.

@Anemy can you take a quick look?

**Comment by Anemy** [comment]
Looks good! :shipit: thanks for the staging links.

**Pull request merged**